### PR TITLE
Updated dependencies to GA versions of the CDK

### PR DESCRIPTION
**Pull request by KrzysztofCwalina**

### CODE REVIEW

File: sdk/provisioning/Azure.Provisioning.CloudMachine/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 ## 1.0.0-beta.2 (Unreleased)
```
Comment (annelo-msft) [inline sdk/provisioning/Azure.Provisioning.CloudMachine/CHANGELOG.md:5]: Nit: need to remove this block, I believe ... or .. are we releasing beta.3 without releasing beta.2?

**Review by christothes — APPROVED** [review]

File: sdk/provisioning/Azure.Provisioning.CloudMachine/CHANGELOG.md (hunk 1 above)
Comment (weshaggard) [reply sdk/provisioning/Azure.Provisioning.CloudMachine/CHANGELOG.md:5]: If we haven't release a beta.2 then the correct thing to do undo the version increment here and in the csproj. Only update the version after we do a release.

**Review by annelo-msft — APPROVED** [review]

**Comment by weshaggard** [comment]
[manually triggered ci job](https://dev.azure.com/azure-sdk/public/_build/results?buildId=4276944&view=results)

**Pull request merged**